### PR TITLE
Assigment2

### PR DESCRIPTION
**Pull request by guilledmv**

### CODE REVIEW

File: Assignment2/guilledmv-Guillermo De Miguel Villanueva.ttl [hunk 1]
```diff
@@ -0,0 +1,15 @@
+@base <http://example.org/class>.
+@prefix components: <http://example.org/class#>.
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+
+:Class01 components: includes: Sensor029;
+			components: includes Computer101
+
+:Sensor029 components: hasMeasurement : Measurement8401.
+
+:Computer101 components: hasOwner : User10A.
```
Comment (ocorcho) [inline Assignment2/guilledmv-Guillermo De Miguel Villanueva.ttl:10]: Parsing problems

**Review by ocorcho — CHANGES_REQUESTED** [review]

**Review by guilledmv — COMMENTED** [review]
I 've corrected some errors

**Review by guilledmv — COMMENTED** [review]
changes

File: Assignment2/guilledmv-Guillermo De Miguel Villanueva.ttl [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+			components: includes :Computer101
+
+:Sensor029 components: hasMeasurement :Measurement8401.
+
+:Measurement8401 components:hasTemperature: "29"^^xsd.integer;
+					components:atTime:"2010-06-12T12:00:12"^^xsd.dateTime.
+
+:Computer101 components: hasOwner :User10A.
+
+:User10A components:hasName :"Pedro"^^xsd.string.
```
Comment (ocorcho) [inline Assignment2/guilledmv-Guillermo De Miguel Villanueva.ttl:16]: Syntax error here.

**Review by ocorcho — CHANGES_REQUESTED** [review]

**Review by guilledmv — COMMENTED** [review]
I tried to correct syntax errors

**Pull request merged**